### PR TITLE
Time conductor Improvement 625

### PR DESCRIPTION
**Pull request by deeptailor**
Implements JPL user request, whereby users would like the bounds of the time conductor to remain the same when switching between UTC based time systems

### CODE REVIEW

File: src/plugins/timeConductor/ConductorTimeSystem.vue [hunk 1]
```diff
@@ -60,7 +60,9 @@ export default {
                 .filter(menuOption => menuOption.clock === (clock && clock.key))
                 .map(menuOption => JSON.parse(JSON.stringify(this.openmct.time.timeSystems.get(menuOption.timeSystem))));
         },
-
+        bothTimeSystemsUtcBased(timeSystem1, timeSystem2) {
```
Comment (psarram) [inline src/plugins/timeConductor/ConductorTimeSystem.vue]: I would change the function name to something like `isTimeSystemsUtcBased()` to better describe what the function does.

Comment (larkin) [reply src/plugins/timeConductor/ConductorTimeSystem.vue]: Inline the check?  it's concise & clear:

```javascript
if (this.bothTimeSystemsUtcBased(this.selectedTimeSystem, timeSystem)) { /*...*/ }
// vs
if (this.selectedTimeSystem.isUTCBased && timeSystem.isUTCBased) { /*...*/ }
```

**Comment by deeptailor** [comment]
@psarram made changes, inlined it like @larkin recommended

File: src/plugins/timeConductor/ConductorTimeSystem.vue (hunk 1 above)
Comment (psarram) [reply src/plugins/timeConductor/ConductorTimeSystem.vue]: Yes, that's even better.

**Pull request merged**